### ISSUE TITLE
Maybe making a project 'public' should require admin confirmation?

### ISSUE THREAD

**Issue by prioux** [aces/cbrain#1045]
I just connected as a demo user and I see all the test public projects that people have created. It's a bit annoying.

![Screen Shot 2020-10-22 at 11 22 23](https://user-images.githubusercontent.com/777588/96893922-380db780-1459-11eb-8040-c4560ff2dbe0.png)

Maybe making a project 'public' should require an administrator to approve? Otherwise we risk everyone's environments to be polluted by tons of public projects created by careless users.

The other possibility is that in the project selection page, we have a button to show/hide public projects.

**Comment by AlexandreHutton** [comment]
Seems like "public" should mean "fetchable." Anybody can make a project public, but other users would have to go find it for it to appear on their public page. So, adding a button like, "browse public projects" followed by "link public project to user project space."

**Comment by MontrealSergiy** [comment]
It is pretty common (though not always the case) data rep there could be review process where somebody approves a submission to some public repository (it is quite lengthy in the case of journals with very involved reviews process editors ets )

**Comment by maxadam2017** [comment]
The projects page has a color key for "Public, User, Personal and Shared projects.  how about making those into radio buttons?  Toggle them on and off to get the view that you like.

**Comment by prioux** [comment]
@AlexandreHutton Yep. But it means we need to save a new piece of 'state' information about each pair [user. project]. I'll think about it.

@maxadam2017 Good suggestion. Complicated to implement (old view code), but it would be nice indeed.

**Comment by MontrealSergiy** [comment]
This new attribute (I would call it endorsed/non-endorsed, vetted, featured) seems to be is easy to add, yet the current panel view should be adjsted by splitting Public into Recomended and Other Public. 

Actually we can have recommended/featured private project, user will see the name of the project in list but would not be able to access it.
